system-variables.md

```diff
@@ -4079,12 +4079,16 @@ As shown in this diagram, when [`tidb_enable_paging`](#tidb_enable_paging-new-in
 
 ### tidb_mpp_store_fail_ttl
 
+> **Warning:**
+>
+> Starting from v9.0.0, this variable is deprecated and its value is fixed to `0s`. This means TiDB no longer waits before sending queries to newly started TiFlash nodes, as the delay is no longer necessary to prevent query failures.
```

Comment:
```suggestion
> Starting from v9.0.0, this variable is deprecated and its value is fixed to `0s`. This means TiDB no longer needs to wait before sending queries to newly started TiFlash nodes, as delays are no longer necessary to prevent query failures.
```